lib/batch/institutional_nucats_cecd.rb

```diff
@@ -0,0 +1,26 @@
+# https://github.com/galterlibrary/digital-repository/issues/678
+# Create new parent collection 'NUCATS Center for Education and Career Development'
+# move NUCATS Grants Repository as child
+
+# NUCATS Grants Repository
+parent_nucats_grants = Collection.find("f2bf6e1d-0e32-4ce2-a52e-bb0522d5708d")
+
+admin_group = 'NUCATS-CECD-System-Admin'
+depositor = 'institutional-nucats-cecd'
```

Comment:
`depositor` isn't used in this script nor is the value `'institutional-nucats-cecd'`, but `parent_nucats_grants.depositor` is used in the other script. Should `depositor` just be removed here or is its value the one that should be used for `nucats_cecd_sys_user.username`?

Reply:
Thanks for catching that. The value is used for converting the collection to institutional.